src/app/features/cases/components/organisms/Workflow/WorkflowStatus.tsx

```diff
@@ -0,0 +1,53 @@
+import React from "react"
+import { Heading, themeColor, themeSpacing } from "@datapunt/asc-ui"
+import styled from "styled-components"
+
+import Table from "app/features/shared/components/molecules/Table/Table"
+
+type CellContent = React.ReactNode
+
+type Props = {
+  status: string    
+  data: Array<Array<CellContent>>
+  showBWVMessage?: boolean
+}
+
+const NewStyledTable = styled(Table)`
```

Comment:
`New`?

Reply:
I didn't get the styling to work, so I tried everything :-) (StyledTable already existed) I will change this.